Gems/Archive/Code/Include/Archive/Clients/ArchiveInterfaceStructs.h

```diff
@@ -299,16 +306,27 @@ namespace Archive
     //! There are 3 blocks per block line as 3 "2 MiB" chunks can be encoded in a 64-bit integer
     //! This is done by storing the compressed block size using 21-bits
     constexpr AZ::u64 BlocksPerBlockLine = 3;
+    //! For block line with a jump entry one of instead of having 3 21-bit compressed block sizes
+    //! one of the block entries is borrowed for the jump table entry
```

Comment:
```suggestion
    //! For a block line with a jump entry of one, instead of having 3 21-bit compressed block sizes
    //! one of the block entries is borrowed for the jump table entry
```